### PR TITLE
Fds 2214 schematic api tests

### PR DESCRIPTION
**Pull request by lakikowolfe**
Improve coverage of schematic API functions

https://sagebionetworks.jira.com/browse/FDS-2214

### CODE REVIEW

File: tests/testthat/test_schematic_rest_api.R [hunk 1]
```diff
@@ -73,74 +160,304 @@ test_that("model_submit successfully uploads to synapse", {
...
+  # bad schema url
+  expect_error(
+    model_component_requirements(
+      url=file.path(schematic_url, "v1/model/component-requirements"),
+      schema_url="https://aaaabad.url.jsonld",
+      source_component="Patient",
+      as_graph = FALSE)
+    )
+  
+  # FIXME
```
Comment (lakikowolfe) [inline tests/testthat/test_schematic_rest_api.R:366]: @afwillia - just wanted to make sure I flagged some notes about behavior I encountered that could probably use some error handling. Not sure how much of a priority this work is though.

File: tests/testthat/test_schematic_rest_api.R [hunk 2]
```diff
@@ -73,74 +160,304 @@ test_that("model_submit successfully uploads to synapse", {
...
+      manifest_id = "syn51078535",
+      access_token = "bad PAT"
+    )
+  )
+})
+
+test_that("get_asset_view_table returns output as expected", {
+  skip_it()
+  
+  # returns manifest from csv
```
Comment (lakikowolfe) [inline tests/testthat/test_schematic_rest_api.R:429]: @afwillia another behavior that made me 🤔 . We could just force the function to use `return_type = json` if that's all thats required rn.

**Comment by lakikowolfe** [comment]
@afwillia did my best to capture the functions as they are.

**Comment by lakikowolfe** [comment]
@afwillia if you click that this has been reviewed I can merge!

**Review by afwillia — APPROVED** [review]

**Pull request merged**